README.md

```diff
@@ -1,3 +1,5 @@
 # project_spring_2020
 
 [![CircleCI](https://circleci.com/gh/biof309/project_spring_2020/tree/master.svg?style=shield)](https://circleci.com/gh/biof309/project_spring_2020/tree/master)
+
+I will do a project
```

Comment:
This is a bit ambitious, but I plan to determine if there is a correlation between BCG vaccination rate (against tuberculosis) and COVID-19 case fatality. To execute this plan, I will find recent data on TB vaccination worldwide (by country) from WHO and recent COVID-19 fatality rates from Jon's Hopkins worldwide by country. I will manipulate this data to be tidy if necessary using the pandas package, and then plot the data using a heatmap, or something of the like, using matplotlib. Then I will create a new data file with case fatality and BCG vaccination rate and perform statistics on this file using numpy, or a similar package.